a3fe/tests/test_run.py

```diff
@@ -348,6 +349,38 @@ def setup_calc(mock_run_process):
             setup_calc.setup(bound_leg_sysprep_config=cfg, free_leg_sysprep_config=cfg)
             yield setup_calc
 
+    @pytest.fixture
+    def setup_calc_with_input_path(self, setup_calc):
```

Comment:
Please just change all of these tmp dirs to used "temp_input_" - this will reduce duplication and speed things up.